test/Microsoft.DotNet.ToolPackage.Tests/Microsoft.DotNet.ToolPackage.Tests.csproj

```diff
@@ -41,8 +41,6 @@
     <None Remove="SampleGlobalTool/**" />
   </ItemGroup>
 
-  <UsingTask TaskName="DownloadFile" AssemblyFile="$(CLIBuildDll)" />
```

Comment:
Not important for this change, move download to test setup instead of msbuild. But there are several merge conflict so I keep it together